sdk_lib/make_chroot.sh

```diff
@@ -47,6 +47,11 @@ DEFINE_string stage3_path "" \
   "Use the stage3 located on this path."
 DEFINE_string cache_dir "" "Directory to store caches within."
 
+DEFINE_string extra_portdir_overlay_path "" \
```

Comment:
Generic is fine, I don't think there is anything else that this would conflict with or be confused with. There used to be other more complicated schemes for pulling in other overlays but that all broke and was pulled out some time ago.